EnglishFilter/sections/specific.txt

```diff
@@ -9769,6 +9769,7 @@ sexcord.com##.spots-box
 ||sexcord.com/sadsrd*.php
 ||sexcord.com/play*.php
 sexlist.tv##.player-ads-side
+sexlist.tv##.mult
```

Comment:
This `||sexlist.tv/myvids/rek/multic.php` should be enough.
Btw, there was a clickunder, I fixed it directly.